app/models/lesson.rb

```diff
@@ -0,0 +1,21 @@
+class Lesson < ApplicationRecord
+  belongs_to :course
+
+  has_one_attached :video
```

Comment:
I think you need to give validation for the type of video you want to allow for the user to upload.

Reply:
Added validations apu. Thank you.